src/declarativedbusinterface.cpp

```diff
@@ -351,7 +352,7 @@ void DeclarativeDBusInterface::typedCall(const QString &method, const QJSValue &
     QDBusPendingCallWatcher *watcher = new QDBusPendingCallWatcher(pendingCall);
     connect(watcher, SIGNAL(finished(QDBusPendingCallWatcher*)),
             this, SLOT(pendingCallFinished(QDBusPendingCallWatcher*)));
-    m_pendingCalls.insert(watcher, callback);
+    m_pendingCalls.insert(watcher, qMakePair(callback, errorCallback));
```

Comment:
But if you check for callability only if the errorCallback is defined? Default constructed values would not be an error, but the callback must be a valid function if it is defined:

if (!errorCallback.isUndefined() && !errorCallback.isCallable()) /* log error */